src/components/text-editor/test-setup/content-generator.ts

```
@@ -0,0 +1,187 @@
+
+/**
+ * Creates a document with a heading.
+ *
+ * @param text - The heading text
+ * @param level - The heading level (1-6)
+ * @param schema - Optional schema to use (defaults to test schema)
+ * @returns An EditorState with a heading
+ */
+export function createDocWithHeading(
+    text: string,
+    level: number = 1,
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)